### PR TITLE
docs: update contributing instructions

### PR DESCRIPTION
**Pull request by Tanuj22**
Signed-off-by: Tanuj22 <tanujagarwal22@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
In the dependencies section, add `npm` too, and change the command for installing NodeJs to use `'npm` instead of brew.
Also, please do add the vs-code signoff image in this branch too.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -91,6 +74,6 @@ Please contribute! The Layer5 site uses GitHub Pages (Jekyll) to host its site.
 `make site` # this will run a local web server with "live reload" conveniently enabled.
 1. Commit and push changes to your remote branch.
 `git push origin <my-username>/<my-changes>`
-1. Open a [pull request](../../pulls) (in your web browser) against the master branch on https://github.com/layer5io/layer5.
+1. Open a [pull request](../../pulls) (in your web browser) against the `layer5-ng` branch on https://github.com/layer5io/layer5.
 
-Build and preview your changes by running `make site` or `make docker`.
```
Comment (leecalcote) [inline CONTRIBUTING.md:96]: Are we able to run the new site in docker?

Comment (Nikhil-Ladha) [reply CONTRIBUTING.md:96]: No, not yet..

Comment (leecalcote) [reply CONTRIBUTING.md:96]: Oh, I see. Ok.

**Comment by Tanuj22** [comment]
> In the dependencies section, add `npm` too, and change the command for installing NodeJs to use `'npm` instead of brew.
> Also, please do add the vs-code signoff image in this branch too.

npm is installed along with nodejs so is it needed seperately? @Nikhil-Ladha

**Review by Nikhil-Ladha — APPROVED** [review]
Ohh, yeah correct. My bad😆

**Pull request merged**